src/Tasks.UnitTests/Exec_Tests.cs

```diff
@@ -48,6 +48,27 @@ private ExecWrapper PrepareExecWrapper(string command)
             return exec;
         }
 
+        [Fact]
+        [Trait("Category", "mono-osx-failing")]
```

Comment:
Why is this failing on other OSes? Does it need fixing there too?

Reply:
Yes, but since we don't currently have a customer asking for it, BenVillalobos wanted to hold off on fixing it until we have feedback that it's needed.